jetstream_pt/third_party/llama/model_exportable.py

```diff
@@ -225,13 +225,12 @@ def get_quantized_embedding_weight_to_scaler_map():
     }
 
   @staticmethod
-  def get_weight_sharding_type():
+  def get_weight_sharding_type(model_name: str = "llama-3"):
```

Comment:
Some potential issue with current logic: 

1: User might think llama-2 is default 
2: If user pass wrong model, we still return a sharding_dict

User may forgot give model_name or give a wrong model_name. I feel it's better to set default as None (or not set default value) and enforce user pass right model name. Have validation in this function and throw error if user didn't pass right model.